src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -14,6 +28,24 @@
 
     @Override
     public int getSize() {
-        return 0;
+        return size;
+    }
+
+    private void resize() {
+        Node<K, V>[] currentTable = table;
+        table = (Node<K, V>[]) new Node[currentTable.length * RESIZE_NUMBER];
+
+    }
+
+    private static class Node<K, V> {
+        final K key;
+        final V value;
+        final Node<K, V> next;
+
+        public Node(K key, V value, Node<K, V> next) {
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
     }
```

Comment:
The `Node` class should be after the public methods of `MyHashMap`. Move the `Node` class to the bottom of the `MyHashMap` class.